KaylaMM/do-you-like-me-2
# Todo2: Preview page dialogue banner

## Changes

A new component was created with chakra UI named HeartDoodleDialogue, as well as a new styling file for that component. Conditional rendering was added on the note page so it only appears on the preview note page.

## Issue ticket number and link
Ticket #[ Todo2](https://github.com/KaylaMM/do-you-like-me-2/issues/3)

## How to test
Use this link, where it should appear: http://localhost:3000/note
Use this link, where it should _not_ appear: http://localhost:3000/note?se=YWZoZGFmaEBhb2wuY29t&sn=d3Jhc2Zo

![Screenshot 2023-11-02 at 12 52 45 PM](https://github.com/KaylaMM/do-you-like-me-2/assets/52683928/cf3f6014-d946-4672-9995-841c561b21d5)

<img width="504" alt="Screenshot 2023-11-02 at 12 54 19 PM" src="https://github.com/KaylaMM/do-you-like-me-2/assets/52683928/8cddf900-674b-43c6-97bf-704860679c47">
